src/NodeAutoCompleteViewExtension/NodeAutoCompleteViewExtension.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Dynamo.Extensions;
+using Dynamo.Logging;
+using Dynamo.Wpf.Extensions;
+
+namespace Dynamo.NodeAutoComplete
+{
+    /// <summary>
+    /// This sample view extension demonstrates a sample IViewExtension 
+    /// which tracks current clicked node port in Dynamo and 
+    /// try to suggest the next best node to connect that port.
+    /// </summary>
```

Comment:
Please replace this with a proper description of the extension.

Reply:
hmm can you be more specific? Any word here not proper? I think this is what we doing for node autocomplete no?

Reply:
hmm, I wouldn't call this extension a sample?

Reply:
Sorry @QilongTang , I guess I read only the first line 😄

Definitely not a sample and not demonstrating anything. The rest is probably OK.

Reply:
Gotcha, will remove sample